doc/releases/changelog-dev.md

```diff
@@ -44,7 +44,43 @@
   [(#2709)](https://github.com/PennyLaneAI/pennylane/pull/2709)
 
 <h3>Improvements</h3>
+  
+* Added the possibility to group samples into counts providing `counts=True` flag in `qml.sample` function. This required
+  creation of a new measurement type `Counts` in `measurements.py`.
+  [(#2686)](https://github.com/PennyLaneAI/pennylane/pull/2686)
+
+  It can be used with states:
+  
+  ```pycon
+  >>> dev = qml.device("default.qubit", wires=2, shots=1000)
 
+  >>> @qml.qnode(dev)
+  >>> def circuit():
```

Comment:
```suggestion
  It can be used when obtaining raw samples in the computational basis:
  
  ```pycon
  >>> dev = qml.device("default.qubit", wires=2, shots=1000)
  >>>
  >>> @qml.qnode(dev)
  >>> def circuit():
```